pkg/controllers/node/termination/suite_test.go

```diff
@@ -186,8 +209,12 @@ var _ = Describe("Termination", func() {
 
 			// Reconcile to delete node
 			node = ExpectNodeExists(ctx, env.Client, node.Name)
-			ExpectObjectReconciled(ctx, env.Client, terminationController, node)
-			ExpectNotFound(ctx, env.Client, node)
+			Eventually(func(g Gomega) {
```

Comment:
Same comment here about Eventually. Do we need to do this with a time-based check or can we get away with doing this in a way that has more determinism around it?